modules/nf-core/bbmap/bbduk/bbmap-bbduk.diff

```diff
@@ -0,0 +1,61 @@
+Changes in module 'nf-core/bbmap/bbduk'
+'modules/nf-core/bbmap/bbduk/meta.yml' is unchanged
+Changes in 'bbmap/bbduk/main.nf':
+--- modules/nf-core/bbmap/bbduk/main.nf
++++ modules/nf-core/bbmap/bbduk/main.nf
+@@ -1,6 +1,6 @@
+ process BBMAP_BBDUK {
+     tag "$meta.id"
+-    label 'process_medium'
++    label 'process_high'
+ 
+     conda "${moduleDir}/environment.yml"
+     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+@@ -12,9 +12,10 @@
+     path contaminants
+ 
+     output:
+-    tuple val(meta), path('*.fastq.gz'), emit: reads
+-    tuple val(meta), path('*.log')     , emit: log
+-    path "versions.yml"                , emit: versions
++    tuple val(meta), path('*.uncontaminated.fastq.gz')  , emit: reads
++    tuple val(meta), path('*.contaminated.fastq.gz')    , emit: contaminated_reads
++    tuple val(meta), path('*.log')                      , emit: log
++    path "versions.yml"                                 , emit: versions
+ 
+     when:
+     task.ext.when == null || task.ext.when
+@@ -23,7 +24,8 @@
+     def args = task.ext.args ?: ''
+     def prefix = task.ext.prefix ?: "${meta.id}"
+     def raw      = meta.single_end ? "in=${reads[0]}" : "in1=${reads[0]} in2=${reads[1]}"
+-    def trimmed  = meta.single_end ? "out=${prefix}.fastq.gz" : "out1=${prefix}_1.fastq.gz out2=${prefix}_2.fastq.gz"
++    def trimmed  = meta.single_end ? "out=${prefix}.uncontaminated.fastq.gz" : "out1=${prefix}_1.uncontaminated.fastq.gz out2=${prefix}_2.uncontaminated.fastq.gz"
++    def contaminated_reads = meta.single_end ? "outm=${prefix}.contaminated.fastq.gz" : "outm=${prefix}_1.contaminated.fastq.gz outm2=${prefix}_2.contaminated.fastq.gz"
```

Comment:
A lot of these feel it should be added to the ofifical bbduk module, `outm` looks like valid bbduk output no?

The uncontaminated/contimated naming is too specific obviously, but the functionality shoiuld go in teh offiical module IMO

Reply:
I will open an issue for detaxizer describing the enhancement of the original module, so that it can be addressed and is not forgotten. Would leave it the way it is for this release, though.